test/parallel/test-process-startTime.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const now = Date.now();
+const diffFromNow = now - process.startTime;
+
+// Make it a bit flexible
+assert.ok(diffFromNow > 0);
```

Comment:
Ah! I don't think it's even theoretically possible, as `process.startTime` is set during initialization of node itself, while `now` is gotten in this test.
Running just this test multiple times locally never gave a lower number than 70ms.